smf/sled-agent/non-gimlet/config-rss.toml

```diff
@@ -97,7 +97,7 @@ infra_ip_last = "192.168.1.30"
 # You can configure multiple uplinks by repeating the following stanza
 [[rack_network_config.uplinks]]
 # The gateway IP for the rack's external network
-gateway_ip = "192.168.1.199"
```

Comment:
Is this your local config? Probably shouldn't be checked-in

Reply:
Good catch, thanks. I keep stashing this, but obviously the muscle memory around `git add -u` is strong :)